libs/agno/agno/db/postgres/schemas.py

```diff
@@ -157,6 +158,20 @@
     "created_at": {"type": String, "nullable": False, "index": True},  # ISO 8601 datetime string
 }
 
+CONFIG_TABLE = {
```

Comment:
I really think it would be better to have an Agent's table with `agent_id` and then a configs table.
The structure below makes it really hard for the following queries:
- Give me the latest config for agent_id `my-agent`
- Roll back `my-agent` to config version 1.2.3